docs/user/04-50-ingress-filtering.md

```diff
@@ -0,0 +1,59 @@
+# Enable Ingress Filtering
+
+> [!NOTE]
+> The traffic filtering feature is only available for SAP internal customers as it is integrated with SAP's geo-blocking solution.
+
+> [!NOTE]
+> The ingress filtering feature is available for the following plans: aws, gcp, azure.
+
+Kyma Environment Broker allows you to either enable or disable an ingress filtering during SAP BTP, Kyma runtime provisioning and update operations.
```

Comment:
In my opinion we should use `provisioning and updating` or `provision and update` to keep it consistent.

Reply:
This is how we call these operations as far as I know: provisioning, update.
See custom networking, custom administrators etc.